opt/start-postgres.sh

```diff
@@ -27,8 +27,6 @@ else
     # Postgresql was probably not shutdown properly. Cleaning up the mess...
     if ! $running ; then
        echo "" > /home/${SYSTEM_USER}/.postgresql/logfile # empty log files
-       rm -vf /home/${SYSTEM_USER}/.postgresql/.s.PGSQL.5432
-       rm -vf /home/${SYSTEM_USER}/.postgresql/.s.PGSQL.5432.lock
        rm -vf /home/${SYSTEM_USER}/.postgresql/postmaster.pid
```

Comment:
Note: I checked and the `postmaster.pid` file is still on the mounted volume (works fine).